app/models/discovery_rule.rb

```diff
@@ -5,6 +5,8 @@ class DiscoveryRule < ActiveRecord::Base
   include Parameterizable::ByIdName
   include Taxonomix
 
+  attr_accessible :name, :search, :hostgroup_id, :hostname, :priority, :organization_ids, :max_count
```

Comment:
I'd also add `enabled`, no need to be private for this one. And also `location_ids` for integrity, although we only test against organizations in our unit tests.